aiden-web/aiden_app/services/chat_service.py

```diff
@@ -14,15 +16,27 @@
 from aiden_app.services.agents.agent import Agent
 from aiden_app.services.tools.utils.cv_editor import CVEditor
 
+from aiden_shared.constants import JOB_COLLECTION
+from aiden_shared.models import JobOffer
+from aiden_shared.utils import reference_to_uuid
+
 
 class ChatService:
     @staticmethod
-    def get_agent_from_session(session):
+    def get_agent_from_session(session) -> MistralAgent:
```

Comment:
since chat service only contains staticmethod or class methods that don't return ChatService instances and ChatService doesn't store any state, why not makin chatservice a simple python module ? I.e. deleting the chatservice class to only keep the functions in this module

Reply:
Yes, it would be better, this is what I did in `chat_service_v2`